### ISSUE TITLE
Nexus configuration script does not wait for API to be live.

### ISSUE THREAD

**Issue by marrobi** [microsoft/AzureTRE#1609]
When running the nexus configuration script soon after deployment the script tries to connect to the Nexus API before it is ready. This is in the time between the password file being created on the file share becoming live.

The script retuens:
```
HTTP/1.1 503 Service Temporarily Unavailable
Content-Length: 19
Server: nginx
Date: Mon, 28 Mar 2022 12:39:56 GMT

curl: (22) The requested URL returned error: 503
```

**Comment by marrobi** [comment]
@jjgriff93 as part of the VM work, is the script going to be run as part of the deployment rather than manually? 

If you can handle this situation would be good.

**Comment by martinpeck** [comment]
This will get resolved as part of #1584
